github/pulls_reviews.go

```diff
@@ -0,0 +1,14 @@
+package github
+
+import "time"
+
+// No docs are out for this quite yet.
+type PullRequestReview struct {
+	ID          *int       `json:"id,omitempty"`
+	User        *User      `json:"user,omitempty"`
+	Body        *string    `json:"body,omitempty"`
+	SubmittedAt *time.Time `json:"submitted_at,omitempty"`
+
+	// State can be "approved", "rejected", or "commented".
+	State *string `json:"state,omitempty"`
+}
```

Comment:
Generally, I would advise against doing something suboptimal now in anticipation of something happening in the future. It's usually better to do the bare minimum for today's needs, and adjust in future as needed. It's very easy to move something into a new file later if there ends up being more stuff later.

But, it's not a big deal, and I see you're at GitHub, so I imagine you have a better ability to anticipate changes. So it's okay with me.